apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/codegen/kotlin/KotlinCodeGen.kt

```diff
@@ -390,19 +393,21 @@ internal object KotlinCodeGen {
   fun TypeRef.asTypeName() = ClassName(packageName = packageName, simpleName = name.capitalize())
 
   private fun Map<String, Any>.toCode(): CodeBlock? {
-    return takeIf { it.isNotEmpty() }?.let {
```

Comment:
here `takeIf` with `?.let` is really hard to understand plus, caused shadowed names since it is nested couple times. I replaced it with a simple if check. Hope it is better.